bedrock/firefox/templates/firefox/sync.html

```diff
@@ -212,9 +214,12 @@
   <div class="inner-wrapper">
     <div class="container">
       <header>
-        <h3>{{ _('Have an <b>Android smartphone</b> or <b>tablet</b>?') }}</h3>
-        <a href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}" class="sync-button green" data-interaction="button click" data-element-action="Sync CTA" id="cta-android-footer">
-          {{ _('DOWNLOAD FIREFOX FOR ANDROID') }}
+        <h3>{{ _('Get Firefox everywhere') }}</h3>
```

Comment:
2 Is not a good idea, we would import the state of that file (e.g. if new.lang is active, we activate sync even if it's not complete).

Given that we already have almost 60 locales, I'll copy that string into a shared file and it will be picked up automatically. So, merge at it is.